test/python/quantum_info/operators/symplectic/test_sparse_pauli_op.py

```diff
@@ -507,6 +507,129 @@ def test_simplify_zero(self, num_qubits):
         np.testing.assert_array_equal(zero_op.paulis.phase, np.zeros(zero_op.size))
         np.testing.assert_array_equal(simplified_op.paulis.phase, np.zeros(simplified_op.size))
 
+    def test_sort(self):
```

Comment:
Could you add a test for a corner case like an empty SparsePauliOp to make sure no errors are being raised?

Reply:
OK. I will add a test.